test/e2e/mv3-perf-stats/bundle-size.js

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env node
+
+/* eslint-disable node/shebang */
+const path = require('path');
+const { promises: fs } = require('fs');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const {
+  isWritable,
+  getFirstParentDirectoryThatExists,
+} = require('../../helpers/file');
+
+const { exitWithError } = require('../../../development/lib/exit-with-error');
+
+/**
+ * The e2e test case is used to capture bundle time statistics for extension.
+ */
+
+const backgroundFiles = [
+  'scripts/runtime-lavamoat.js',
+  'scripts/lockdown-more.js',
+  'scripts/sentry-install.js',
+  'scripts/policy-load.js',
+];
+
+const uiFiles = [
+  'scripts/sentry-install.js',
+  'scripts/runtime-lavamoat.js',
+  'scripts/lockdown-more.js',
+  'scripts/policy-load.js',
+];
+
+const BackgroundFileRegex = /background-[0-9]*.js/u;
+const CommonFileRegex = /common-[0-9]*.js/u;
+const UIFileRegex = /ui-[0-9]*.js/u;
+
+async function main() {
+  const { argv } = yargs(hideBin(process.argv)).usage(
+    '$0 [options]',
+    'Run a page load benchmark',
```

Comment:
Pre-existing erroneous description:
```suggestion
    'Capture bundle size stats',
```